Sprint-3/4-stretch-investigate/card-validator.js

```diff
@@ -0,0 +1,55 @@
+function cardNumberValidator(cardNumber) {
+  if (cardNumber.length !== 16) {
+    return false;
+  }
+  const numberInArray = cardNumber.toString().split(""); // convert the cardNumber to sting then into an array
+  // check if all numbers are numbers
+  let isAllNumbers = true;
+  for (let index = 0; index <= numberInArray.length - 1; index++) {
+    const currentNumber = numberInArray[index];
+    if (isNaN(currentNumber)) {
+      isAllNumbers = false;
+      break;
```

Comment:
What's the benefit of setting a boolean and breaking, rather than just writing `return false;` here?